simple_network_sim/network_of_populations.py

```diff
@@ -160,14 +242,23 @@ def doIncomingInfectionsByNode(graph, currentState, movementMultiplier):
 
 
 def getWeight(graph, orig, dest, multiplier):
+    """Get the weight of the edge from orig to dest in the graph. If the edge doesn't have a
+    weight, 1.0 is assumed and the returned weight is adjusted by the multiplier and any
+    delta_adjustment on the edge.
```

Comment:
I think this function name is a bit too generic, but maybe worth mentioning in the docs that this weight is expected to be proportional to the movement between nodes.